### PR TITLE
Image component

### PR DESCRIPTION
**Pull request by abhisheksahani-nova**

### CODE REVIEW

**Review by vrinda-mahajan — APPROVED** [review]
LGTM👍

**Comment by abhisheksahani-nova** [comment]
> LGTM👍

Thanks for review

File: Components/image/image.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.res-img {
+  width: 100%;
+  height: auto;
+  object-fit: cover;
+}
+
+.sq-img {
+  width: 11rem;
+  height: 11rem;
+  object-fit: cover;
```
Comment (anik31) [inline Components/image/image.css:10]: Instead of declaring object-fit:cover separately in all classes you can add it commonly.

Comment (abhisheksahani-nova) [reply Components/image/image.css:10]: I can but for one css property making a class is not worth it , as Vikram also said in the class

**Review by partha8 — APPROVED** [review]
looks good

**Pull request merged**